contracts/0.4.24/StETH.sol

```diff
@@ -81,6 +81,15 @@ contract StETH is IERC20, Pausable {
      */
     bytes32 internal constant TOTAL_SHARES_POSITION = keccak256("lido.StETH.totalShares");
 
+    /**
+      * @notice An executed shares transfer from `sender` to `recipient`
+      */
+    event TransferShares(
```

Comment:
We define other events in interfaces, e.g. `Stopped` and `Resumed` are defined in `ISTETH.sol`. I think this event should also be moved there for consistency.

Reply:
Just did it properly, thx!